### ISSUE TITLE
Issue: Search by opponent's name

### ISSUE THREAD

**Issue by Paula94** [Epix37/Hearthstone-Deck-Tracker#975]
Hi,
-----

tl;dr - How do you search on 'stats' for a previous played game by filtering using the opponent's name?

------

Re - my question: I've tried 'stats>overall>games' and setting the 'note filter' to the player name. This only works if I've manually entered the player name into the 'note' field after I originally played them, (but it's already recorded this data in the 'name' field so I didn't repeat it.)

At the moment I go to 'stats>overall>games' page, and sort alphabetically by the 'name' column, but my previous games list is now too long for this to be practical.

Searching by name would be a very useful feature in this excellent app, as I'm now starting to come up against the same opponents and I need to check if they're ZooLock or HandLock players. :)

Thanks for any help you can give, and thanks for your hard work on this app.
Paula

**Comment by culinko** [comment]
Currently, there is no filter for opponent names. Somebody already suggested some new filters including yours in #692.

**Comment by Epix37** [comment]
Closing in favor of #692
